group_vars/all

```diff
@@ -69,9 +74,9 @@ pkg_version_pydm_opi: "v1.5.7"
 pkg_version_mathphys: "v2.0.0"
 pkg_version_trackcpp: "v4.1.0"
 pkg_version_lnls: "v1.0.0"
-pkg_version_pyaccel: "v3.1.1"
+pkg_version_pyaccel: "v3.2.0"
 pkg_version_pymodels: "v1.2.0"
-pkg_version_apsuite: "v2.8.0"
+pkg_version_apsuite: "v2.9.0"
 
 pkg_version_linac_ioc_ps: "v1.2.0"
 pkg_version_linac_opi: "v1.1.3"
```

Comment:
@xresende, why is this variable doubled here (look at line 59)?

Reply:
problem introduced with merge! fixed.